activities/Speak.activity/css/activity.css

```diff
@@ -42,13 +42,67 @@
 #userText {
 	height:40px;
 	margin:5px;
-	margin-left:3vh;
+	margin-left:8vh;
 	width:90vw;
 	font-size: 28pt;
 	font-weight: bold;
 	z-index:0;
 }
 
+
+/** Custom dropdown arrow for select tag **/
+
+.dropdown {
+  position: absolute;
+}
+
+.dropdown:before {
+  content: "";
+  position: absolute;
+	margin-left: 5px;
+  left: 23px;
+  top: 27px;
+	width: 0;
+	height: 0;
+	border-left: 10px solid transparent;
+	border-right: 10px solid transparent;
+	border-top: 10px solid #000;
+  pointer-events: none;
+}
+
+/** CSS for the select tag to make it look like a button **/
+
+#combo-box {
+  position:absolute;
+  top: 5px;
+  width:2vw;
+  height:25px;
+  line-height:20px;
+  margin:13px;
+  margin-left:3vh;
+  padding:0;
+  font-size: 0;
```

Comment:
In Chrome, this causes the actual drop-down text to not appear as well. To fix add a new CSS selector with a normal font size. (see here: https://github.com/llaske/sugarizer/pull/250/files#diff-6b8bcb22150cf3bd907abf1eaa17ba77R170)
![demo](https://i.imgur.com/POTu6w0.gif)